ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/PrivacyBlockProcessor.java

```diff
@@ -51,4 +123,37 @@ public Result processBlock(
         .commit();
     return blockProcessor.processBlock(blockchain, worldState, blockHeader, transactions, ommers);
   }
+
+  protected void rehydratePrivacyGroupHeadBlockMap(
```

Comment:
It looks like this method isn't being used. Did we forget to use it or should we remove it?

Reply:
yes - this is moved inside PrivateStateRehydration

Reply:
fixed